jsonwebtoken/jsonwebtoken.github.io
# Feat: Add persistence to the ad banner with local storage

## Description

This PR does the following:
- Add persistence of the state for the ad banner through local storage, so when users close the ad banner for the current campaign value defined by the `messageBar.id.value` property, it won't show the ad banner when users revisit the page.
- Updates the copy for the ad banner to promote the best of devday.

## How to test it 
- Navigate to the home page and verify that it loads the ad banner. Then, click on the close icon and refresh the page. Verify that it doesn't show the ad banner again.
